File: app/src/main/java/com/demo/demoapplication/ui/utils/ImageUtil.java
package com.demo.demoapplication.ui.utils;

import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.util.Base64;

import java.io.ByteArrayOutputStream;

public class ImageUtil {

    public static Bitmap decodeImage64(String base64Str) throws IllegalArgumentException {

        byte[] bytes=Base64.decode(base64Str,Base64.DEFAULT);
        Bitmap bitmap= BitmapFactory.decodeByteArray(bytes,0,bytes.length);

        return bitmap;
    }

    public static String encodeImage64(Bitmap bitmap) {
        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
        bitmap.compress(Bitmap.CompressFormat.PNG, 100, outputStream);

        return Base64.encodeToString(outputStream.toByteArray(), Base64.DEFAULT);
    }
}

File: app/src/main/java/com/demo/demoapplication/ui/utils/ImageUtil.kt
package com.demo.demoapplication.ui.utils

import kotlin.Throws
import android.graphics.Bitmap
import android.graphics.BitmapFactory
import android.util.Base64
import java.io.ByteArrayOutputStream
import java.lang.IllegalArgumentException

object ImageUtil {
    @Throws(IllegalArgumentException::class)
    fun decodeImage64(base64Str: String?): Bitmap {
        val bytes =
            Base64.decode(base64Str, Base64.DEFAULT)
        return BitmapFactory.decodeByteArray(bytes, 0, bytes.size)
    }

    fun encodeImage64(bitmap: Bitmap): String {
        val outputStream = ByteArrayOutputStream()
        bitmap.compress(Bitmap.CompressFormat.PNG, 100, outputStream)
        return Base64.encodeToString(outputStream.toByteArray(), Base64.DEFAULT)
    }
}